.github/workflows/build.yml

```
@@ -0,0 +1,18 @@
+name: build the package
+on:
+  pull_request:
```

```suggestion
  pull_request:
    - main
```

Maybe this will trigger the workflow?